src/features/react-virtualized-renderers/flowTypes.js

```diff
@@ -0,0 +1,154 @@
+// @flow
+import { SortDirection as SORT_DIRECTION } from 'react-virtualized/dist/es/Table/index';
+
+import type { Node } from 'react';
+import { TYPE_FOLDER, TYPE_FILE, TYPE_WEBLINK } from '../../constants';
+
+const { ASC, DESC } = SORT_DIRECTION;
+
+type SortDirection = ASC | DESC;
+
+type RowRendererParams = {
+    className: string,
+    columns: Array<any>,
+    index: number,
+    isScrolling: boolean,
+    onRowClick: ?Function,
+    onRowDoubleClick: ?Function,
+    onRowMouseOut: ?Function,
+    onRowMouseOver: ?Function,
+    rowData: any,
```

Comment:
Can we avoid use of `any` in these types?

Reply:
Note: change the declaration at the top of the file to `// @flow strict` to enforce this and the function type

Reply:
File another Enhancement ticket for this issue: https://jira.inside-box.net/browse/COXP-7149

Reply:
Filed another enhancement ticket for this issue.